net/pfSense-pkg-frr/files/usr/local/pkg/frr/inc/frr_bgp.inc

```diff
@@ -967,4 +967,3 @@ function frr_generate_config_bgp() {
 		$frr_integrated_config['rpki'] = frr_bgp_generate_rpki_servers();
 	}
 }
- 
```

Comment:
The removal of EOL from last line created a conflict.  Please revert this

Reply:
That's strange I've got a newline in my local (up to date with github) branch and I certainly don't remember removing it consciously. I'll sort it out.

Reply:
Hmm that's officially strange, looks like git(hub) is removing the newlines (from display at least) even though they're present in my local copy. Rebase fixed it.

Reply:
There wasn't a newline that got removed, there was an odd character on a line at the end of that file that broke the package. I removed it yesterday in f704edf8024c3ed86d8f642d4c3618930ebd6495

Reply:
See also: https://redmine.pfsense.org/issues/11011